Sources/PackageDescription4/Package.swift

```diff
@@ -13,12 +13,13 @@ import Glibc
 #else
 import Darwin.C
 #endif
+import Foundation
 
 /// The description for a complete package.
 public final class Package {
 
     /// Represents a package dependency.
-    public class Dependency {
+    public final class Dependency: Encodable {
```

Comment:
I would prefer to keep only Encodable related changes in this PR. Can you propose the "final" diff separately?